### PR TITLE
Tp6m

### PR DESCRIPTION
**Pull request by klahnakoski**
This shows the pageload (and other suites of a single test) 

Requires https://github.com/mozilla-frontend-infra/firefox-health-dashboard/pull/291

### CODE REVIEW

**Comment by klahnakoski** [comment]
Not including #291 and not including perf-goggles add

https://github.com/mozilla-frontend-infra/firefox-health-dashboard/pull/308/files/0ce79325e1544040596b5def8080c82e1790c15f..2a705889ade0f22f57037962c2882368a04eaeec?utf8=%E2%9C%93&diff=split&w=1

**Comment by klahnakoski** [comment]
Adds a mobile page:

http://localhost:5000/quantum/tp6m?test=loadtime

Also allows the TP6 page to specify which (sub)test to display

http://localhost:5000/quantum/tp6?bits=64&test=loadtime

File: src/containers/PerfherderGraphContainer/index.jsx [hunk 1]
```diff
@@ -65,7 +65,7 @@ PerfherderGraphContainer.propTypes = {
       seriesConfig: PropTypes.shape({
         extraOptions: PropTypes.arrayOf(PropTypes.string),
         frameworkId: PropTypes.number.isRequired,
-        option: PropTypes.string.isRequired,
+        option: PropTypes.string,
```
Comment (helfi92) [inline src/containers/PerfherderGraphContainer/index.jsx]: For none required properties, define a default prop. Example:

```jsx
PerfherderGraphContainer.defaultProps = {
  option: null,
  // ...
}
```

File: src/quantum/index.jsx [hunk 2]
```diff
@@ -98,7 +98,7 @@ export default class QuantumIndex extends React.Component {
       },
       {
         title: 'Page Load tests (TP6)',
-        more: `/quantum/tp6?bits=${bits}`,
+        more: `/quantum/tp6?bits=${bits}&test=pageload`,
```
Comment (helfi92) [inline src/quantum/index.jsx]: https://deploy-preview-308--firefox-health-dashboard.netlify.com/quantum/tp6?bits=64&test=pageload shows:

screen shot 2019-02-20 at 8 45 22 am

File: src/routes.jsx [hunk 3]
```diff
@@ -7,6 +7,7 @@ import JsTeam from './views/JsTeam';
 import NimbledroidGraphPage from './views/NimbledroidGraph';
 import Quantum from './quantum/index';
 import TP6 from './views/QuantumTP6/index';
+import TP6M from './views/QuantumTP6m/index';
```
Comment (helfi92) [inline src/routes.jsx]: No need to add `/index` at the end: `import TP6M from './views/QuantumTP6m';`. The index file is the default import.

File: src/utils/perfherder/chartJs/perfherderFormatter.js [hunk 4]
```diff
@@ -8,12 +8,16 @@ const dataToChartJSformat = data =>
     y: value,
   }));
 const generateInitialOptions = series => {
-  const higherIsBetter = !series.meta.lower_is_better;
+  // TODO: map tests and suite scores to measurement units and
+  // add some label for scale
+  const isTest = !!series.meta.test;
```
Comment (helfi92) [inline src/utils/perfherder/chartJs/perfherderFormatter.js]: `const isTest = Boolean(series.meta.test);`

File: src/quantum/index.jsx (hunk 2 above)
Comment (klahnakoski) [reply src/quantum/index.jsx]: thanks for catching that!

File: src/containers/PerfherderGraphContainer/index.jsx (hunk 1 above)
Comment (klahnakoski) [reply src/containers/PerfherderGraphContainer/index.jsx]: This is more involved

Comment (klahnakoski) [reply src/containers/PerfherderGraphContainer/index.jsx]: https://github.com/mozilla-frontend-infra/firefox-health-dashboard/issues/312

**Comment by klahnakoski** [comment]
@helfi92   one more review, since you caught mistakes in the last one, and I added links to the tp6m page

here are the changes since last review:

https://github.com/mozilla-frontend-infra/firefox-health-dashboard/pull/308/files/dc4988f4022b87eae7df1c4a968a9072fe78f9d2..4bdbce7ac5f670566e748666bdd1304f9658a286?utf8=%E2%9C%93&diff=split&w=1

File: src/routes.jsx [hunk 5]
```diff
@@ -6,8 +6,8 @@ import AndroidPage from './views/Android';
 import JsTeam from './views/JsTeam';
 import NimbledroidGraphPage from './views/NimbledroidGraph';
 import Quantum from './quantum/index';
```
Comment (helfi92) [inline src/routes.jsx]: `import Quantum from './quantum';`

Comment (klahnakoski) [reply src/routes.jsx]: https://deploy-preview-308--firefox-health-dashboard.netlify.com/quantum/tp6?bits=64&test=pageload  is expected to fail like this:  The URL has been fixed.

Concerning `import Quantum from './quantum/index';`  does your linter catch this?  Mine does not.

Comment (helfi92) [reply src/routes.jsx]: I saw this when looking at the diff. Netlify previews ease the process of reviews such that I don't always need to pull the code down :)

**Pull request merged**